pkg/runtime/dispatcher.go

```diff
@@ -219,6 +219,13 @@ func (m *dispatcher) Check(ctx context.Context, requestBag attribute.Bag) (*adap
 		},
 	)
 	res, _ := cres.(*adapter.CheckResult)
+	if res == nil { // There was nothing to do.
+		res = &adapter.CheckResult{}
+	}
+
+	if glog.V(3) {
+		glog.Infof("Check %s", res)
```

Comment:
If using `Infof`, `glog.V(3).Infof(...)` should be better (shorter and clearer, I think)